### ISSUE TITLE
Action buttons in patterns do not have any visual indication that they are actionable items

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#872]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode
- High Contrast ```#1```
- High Contrast ```#2```

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![19 1](https://user-images.githubusercontent.com/22068434/88356701-91acfb80-cd1d-11ea-8526-06bcd416b7c0.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release found at https://github.com/Microsoft/accessibility-insights-windows/releases. Please validate and close at your convenience. Thanks!

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1388.01. All 6 modes are now compliant
